Dockerfile

```diff
@@ -20,6 +20,7 @@ WORKDIR /app/backend
 
 COPY backend/package.json .
 COPY backend/package-lock.json .
+COPY backend/config/ config/
```

Comment:
Not sure if this is correct. Build was complaining of this missing, but if the image was building previously, then it might be a non-issue? I believe it is not being used since we are using database strings for sequelize.

Reply:
Could you attach the exact message? That's a kinda weird because there's no config directory there.

Reply:
`ERROR: Cannot find "/app/backend/config/config.json". Have you run "sequelize init"?`

I get this error when running `docker run -it call-home`.

Reply:
Strange, this should be handled by the `.sequelizerc`. Not sure why this fails locally for me too. Let's remove this line for now, and we'll add a ticket to the backlog to look at this.

Reply:
added ticket. we should add a label for non feature related engineering tickets to notion.